ScilifelabDataCentre/dds_web
# Add troubleshooting to web

# Description

Troubleshooting document collected from Confluence page and displayed. The layout etc should of course be changed and when we work on the web more we can improve on it, but this is a good start and lets us have all information on confluence and just get it to the web page when needed. 

The same thing should be done with the technical overview. 

**To test that the cache works, manually:** 
1. Start `docker` with `full-dev` mode
2. Change the cache `ttl` from `hours=4` to `seconds=30` 
3. Go to dds home page and then click the Troubleshooting button in the navbar
4. Go to Confluence and change something at the top of that page (+ publish)
5. Go to the troubleshooting page again and update
6. Repeat 5 a couple of times
7. (Please change the confluence page back to what it was before the change)

This should mean that the troubleshooting guide should display the exact same thing for 30 seconds, and after that (when you update the page), the recent change should be visible. 

- [x] Summary of the changes and the related issue
- [x] Relevant motivation and context
- [ ] Any dependencies that are required for this change

Fixes DDS-1321

## Type of change


# Checklist:

Please delete options that are not relevant.

- [x] Rebase/merge the branch which this PR is made to
- [x] Changes to the database schema: A new migration is included in the PR
- [ ] Product Owner / Scrum Master: This PR is made to the `master` branch and I have updated the [version](../dds_web/version.py)
- [ ] I am bumping the major version (e.g. 1.x.x to 2.x.x) and I have made the corresponding changes to the CLI version

## Formatting and documentation

- [x] I have added a row in the [changelog](../CHANGELOG.md)
- [x] The code follows the style guidelines of this project: Black / Prettier formatting

## Tests



<a href="https://gitpod.io/#https://github.com/ScilifelabDataCentre/dds_web/pull/1244"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
